src/style.css

```diff
@@ -1,3 +1,81 @@
-body {
+:root {
+  --nav-text: #000;
+  --nav-hover: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+li,
+a {
+  text-decoration: none;
+  color: var(--nav-text);
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.nav__links li a {
+  transition: all 0.3s ease 0s;
+}
+
+.nav__links li a:hover {
+  color: var(--nav-hover);
+}
+
+.header {
+  width: 100vw;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+li {
+  position: relative;
+  display: inline-block;
+  text-transform: uppercase;
+  padding: 0 10px;
+}
+
+li:first-of-type {
+  padding-left: 0;
+}
+
+li:last-of-type {
+  padding-right: 0;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  height: 4px;
+  background-color: var(--nav-hover);
+  border-radius: 50;
```

Comment:
The `border-radius` property should have a unit specified. It currently has a value of `50`, which is likely intended to be `50%` to create a rounded effect. Please correct this to ensure the desired styling is applied.